lib/std/setutils.nim

```diff
@@ -7,26 +7,31 @@
 #    distribution, for details about the copyright.
 #
 
-## This module adds functionality to the built-in `set` type.
-## See also std/packedsets, std/sets
+## This module adds functionality for the built-in `set` type.
+##
+## See also
+## ========
+## * `std/packedsets <packedsets.html>`_
+## * `std/sets <sets.html>`_
 
-import typetraits
+import std/typetraits
 
 #[
   type SetElement* = char|byte|bool|int16|uint16|enum|uint8|int8
     ## The allowed types of a built-in set.
 ]#
 
 template toSet*(iter: untyped): untyped =
-  ## Return a built-in set from the elements of iterable `iter`
-  runnableExamples: 
+  ## Returns a built-in set from the elements of the iterable `iter`.
+  runnableExamples:
     assert "helloWorld".toSet == {'W', 'd', 'e', 'h', 'l', 'o', 'r'}
     assert toSet([10u16, 20, 30]) == {10u16, 20, 30}
     assert [30u8, 100, 10].toSet == {10u8, 30, 100}
     assert toSet(@[1321i16, 321, 90]) == {90i16, 321, 1321}
     assert toSet([false]) == {false}
     assert toSet(0u8..10) == {0u8..10}
+
   var result: set[elementType(iter)]
   for x in iter:
-    incl(result, x)
+    result.incl(x)
```

Comment:
keep this as before; MCS/UFCS doesn't work reliably inside templates (see manual): this doesn't give an opensymchoice